### ISSUE TITLE
[BUG] low quality compose file parsing errors

### ISSUE THREAD

**Issue by brandonbloom** [deref/exo#228]
```
  zookeeper:
    image: confluentinc/cp-zookeeper:5.5.0
    #<<: *defaults
    #mem_limit: 500m
    hostname: zookeeper
    ports:
      - "2181:2181"
    environment:
      ZOOKEEPER_CLIENT_PORT: "2181"
      ZOOKEEPER_TICK_TIME: "2000"
      KAFKA_HEAP_OPTS: '-Xms256M -Xmx256M'
      KAFKA_OPTS: "-Dzookeeper.4lw.commands.whitelist=*"
    healthcheck:
      test: 'echo "ruok" | nc -w 2 -q 2 localhost 2181 | grep imok'
exo apply docker-compose.yml
Error: parsing: 1:13: | is not a valid word
parsing: 1:13: | is not a valid word
```

**Comment by kendru** [comment]
@BenElgar is this solved with your work on #217?

**Comment by BenElgar** [comment]
Yes, hopefully. I'll assign to myself and close when fixed.
